Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -19,465 +19,555 @@
+    {{201, 0x2000}, {"Hidden Treasure", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{202, 0x2000}, {"Healing Elixer", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{203, 0x2000}, {"Ghost Pirate Swords", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{204, 0x2000}, {"Time Twister Hourglass", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{205, 0x2000}, {"Sky-Iron Shield", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{206, 0x2000}, {"Winged Boots", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{207, 0x2000}, {"Sparx Dragonfly", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{208, 0x1206}, {"Dragonfire Cannon", Game::Giants, Element::Other, Type::Item}},
+    {{208, 0x1602}, {"Golden Dragonfire Cannon", Game::Giants, Element::Other, Type::Item}},
+    {{209, 0x1206}, {"Scorpion Striker Catapult", Game::Giants, Element::Other, Type::Item}},
+    {{230, 0x0000}, {"Hand Of Fate", Game::TrapTeam, Element::Other, Type::Item}},
+    {{230, 0x3403}, {"Hand Of Fate (Legendary)", Game::TrapTeam, Element::Other, Type::Item}},
+    {{231, 0x0000}, {"Piggy Bank", Game::TrapTeam, Element::Other, Type::Item}},
+    {{232, 0x0000}, {"Rocket Ram", Game::TrapTeam, Element::Other, Type::Item}},
+    {{233, 0x0000}, {"Tiki Speaky", Game::TrapTeam, Element::Other, Type::Item}},
+    {{300, 0x0000}, {"Dragon's Peak", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{301, 0x2000}, {"Empire Of Ice", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{302, 0x2000}, {"Pirate Ship", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{303, 0x2000}, {"Darklight Crypt", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{304, 0x2000}, {"Volcanic Vault", Game::SpyrosAdv, Element::Other, Type::Item}},
+    {{305, 0x3000}, {"Mirror Of Mystery", Game::TrapTeam, Element::Other, Type::Item}},
+    {{306, 0x3000}, {"Nightmare Express", Game::TrapTeam, Element::Other, Type::Item}},
+    {{307, 0x3206}, {"Sunscraper Spire", Game::TrapTeam, Element::Other, Type::Item}},
+    {{308, 0x3206}, {"Midnight Museum", Game::TrapTeam, Element::Other, Type::Item}},
+    {{3200, 0x2000}, {"Battle Hammer", Game::SwapForce, Element::Other, Type::Item}},
+    {{3201, 0x2000}, {"Sky Diamond", Game::SwapForce, Element::Other, Type::Item}},
+    {{3202, 0x2000}, {"Platinum Sheep", Game::SwapForce, Element::Other, Type::Item}},
+    {{3203, 0x2000}, {"Groove Machine", Game::SwapForce, Element::Other, Type::Item}},
+    {{3204, 0x2000}, {"Ufo Hat", Game::SwapForce, Element::Other, Type::Item}},
+    {{3300, 0x2000}, {"Sheep Wreck Island", Game::SwapForce, Element::Other, Type::Item}},
+    {{3301, 0x2000}, {"Tower Of Time", Game::SwapForce, Element::Other, Type::Item}},
+    {{3302, 0x2206}, {"Fiery Forge", Game::SwapForce, Element::Other, Type::Item}},
+    {{3303, 0x2206}, {"Arkeyan Crossbow", Game::SwapForce, Element::Other, Type::Item}},
+    {{210, 0x3001}, {"Magic Log Holder", Game::TrapTeam, Element::Magic, Type::Trap}},
```

Comment:
Ditto with the order by id comment

Reply:
See comment on previous id order comment